customer-commons/src/test/resources/features/get_customer.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Get Customer
+
```

Comment:
You can apply a background definition

c.f. https://cucumber.io/docs/gherkin/reference/#background